atlp-rwanda/stackup2-barefoot-frontend
# #171380698 Implement requests table

#### What does this PR do?

- Implement the requests table
- Fix the testimonials section of the landing page

#### Description of Task to be completed?
- Install required libraries
- Implement requests table
- Write tests
- Implement carousel for testimonials

#### How should this be manually tested?
- In your terminal, run `git clone https://github.com/Stackup-Rwanda/stackup2-barefoot-frontend.git` to clone this repo

- Run `git fetch origin ft-implement-requests-table-171380698:ft-implement-requests-table-171380698` to pull this PR's branch

- Run `git checkout ft-implement-requests-table-171380698` to switch to the fetched branch above

- Run `yarn install` to install all dependencies

- Start development server by running `yarn dev`.

- The server should launch the app in your default browser at `localhost:{{PORT}}`

- Click on the `Create account` in the navbar to navigate to the requests page

- **If credentials are valid and you have made some requests**
![Screenshot from 2020-06-11 11-15-18 1](https://user-images.githubusercontent.com/45935765/84378359-57abee00-abe4-11ea-90b8-8b3be9dc104c.png)

- **If credentials are valid but you haven't yet made any requests**
![Screenshot from 2020-06-11 12-19-00 1](https://user-images.githubusercontent.com/45935765/84378324-45ca4b00-abe4-11ea-9545-a48b91679fea.png)


- **Invalid credentials**
![Screenshot from 2020-06-11 12-49-21 1](https://user-images.githubusercontent.com/45935765/84378291-33501180-abe4-11ea-852a-0040618dc01c.png)


- Testimonials carousel
![Screenshot from 2020-06-11 12-04-10 1](https://user-images.githubusercontent.com/45935765/84378235-1c112400-abe4-11ea-80e1-0457495b1a28.png)


#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
  [#171380698](https://www.pivotaltracker.com/story/show/171380698)
